app/models/manageiq/providers/embedded_ansible/automation_manager/playbook_runner.rb

```diff
@@ -173,4 +180,13 @@ def delete_job_template
     _log.log_backtrace(err)
     false
   end
+
+  def log_stdout(succeeded)
```

Comment:
It's hard to know what is required as an argument to this method without knowing its internal workings.  Should this be called `log_stdout?` with the argument having a default?  In the current format you could pass a `1` or `yes` as an argument and it would still work. But passing `no` would also be treated as a `true`  Is that expected behavior?